docs/types.rst

```diff
@@ -96,7 +96,23 @@ In Solidity, division rounds towards zero. This mean that ``int256(-5) / int256(
 Note that in contrast, division on :ref:`literals<rational_literals>` results in fractional values
 of arbitrary precision.
 
-Division by zero and modulus with zero throws a runtime exception.
+.. note::
+  Division by zero causes a failing assert.
+
+Modulo
```

Comment:
While this was changed to modulo the summary table above still uses `% (remainder)`